.github/workflows/codingStandards.yml

```diff
@@ -7,11 +7,11 @@ on:
   pull_request: { }
 
 jobs:
-  phpstan:
+  phpstan7:
     name: PHPStan
```

Comment:
Can you please identify the job name :D 

E.g. PHPStan 7.4, same for the other jobs